akvo/rest/serializers/user.py

```diff
@@ -145,17 +142,14 @@ def validate(self, data):
             raise serializers.ValidationError(_('Passwords do not match.'))
 
         password = data['new_password1']
-        check_password_minimum_length(password)
-        check_password_has_number(password)
-        check_password_has_upper(password)
-        check_password_has_lower(password)
-        check_password_has_symbol(password)
+        validate_password(password, self.instance)
 
         return data
 
-    def update(self, instance, validated_data):
-        instance.set_password(validated_data.get('new_password2', instance.password))
-        return instance
+    def save(self):
```

Comment:
is there a reason for using `save` instead of `update`?

Reply:
I want the instance to be saved first before being returned. The `save` method is more idiomatic in that regard and the code calling it is also simpler IMO.